### ISSUE TITLE
v6.0.7 release is broken for cmake users: 

### ISSUE THREAD

**Issue by fduncanh** [nodejs/llhttp#166]
CMake Error at CMakeLists.txt:81 (add_library):
  Cannot find source file:

    llhttp-6.0.7/src/llhttp.c

we need a release like llhttp-6.0.6
```
unzip llhttp-release-v6.0.6.zip
Archive:  llhttp-release-v6.0.6.zip
4bfeb625ce4961bd539c95593529936503be7db4
   creating: llhttp-release-v6.0.6/
  inflating: llhttp-release-v6.0.6/CMakeLists.txt  
  inflating: llhttp-release-v6.0.6/LICENSE-MIT  
  inflating: llhttp-release-v6.0.6/README.md  
  inflating: llhttp-release-v6.0.6/common.gypi  
   creating: llhttp-release-v6.0.6/include/
  inflating: llhttp-release-v6.0.6/include/llhttp.h  
  inflating: llhttp-release-v6.0.6/llhttp.gyp  
   creating: llhttp-release-v6.0.6/src/
  inflating: llhttp-release-v6.0.6/src/api.c  
  inflating: llhttp-release-v6.0.6/src/http.c  
  inflating: llhttp-release-v6.0.6/src/llhttp.c          <--------------------------missing in 6.0.7!!
```

**Comment by fduncanh** [comment]
@mcollina 
I think you also released v6.0.6

please could you provide  a similar llhttp-release-v6.0.7.[zip,tgz] for cmake users that also has
llhttp.c in src/

(I'm guessing this is somehow automatically generated, since I can't see it in the github repo.)


Thank you for maintaining llhttp!

**Comment by mcollina** [comment]
Where did you download that bundle file?

**Comment by fduncanh** [comment]
https://github.com/nodejs/llhttp/releases

the llhttp-release-v6.0.6 is available there for cmake (c-language) users
(I think you released it)

llhttp is  included with https://github.com/FDH2/UxPlay which I maintain.

I would upgrade the included llhttp to 6.0.7 if llhttp.c is available.

**Comment by fduncanh** [comment]
@mcollina 

I'm not sure what the difference between 2.1.5 and 6.0.7 is,  but it seem you have released the wrong code as 6.0.7,  

see
https://github.com/nodejs/llhttp/compare/v6.0.7...release/v6.0.6

**Comment by mcollina** [comment]
Fixed, thanks!
